app/controllers/npq_separation/admin/finance/statements/stale_controller.rb

```diff
@@ -0,0 +1,5 @@
+class NpqSeparation::Admin::Finance::Statements::StaleController < NpqSeparation::AdminController
+  def index
+    @statements = Statement.with_delayed_authorisations
```

Comment:
Is there a reason not to use Statements::Query like `PaidController` does

Also I'm maybe missing something but you've got the pagination helper in the view but nothing seems to be passing the page into the above scope?